_source/about/index.html

```diff
@@ -10,26 +10,25 @@
 {%- capture md -%}
 
 
-Academy of Art and Design Basel
-Visual Communication Institute / The Basel School of Design,
+[Academy of Art and Design Basel](https://www.fhnw.ch/de/die-fhnw/hochschulen/hgk)  
+[Visual Communication Institute / The Basel School of Design](http://thebaselschoolofdesign.ch),  
 Basel, Switzerland
 
-A project of the [Interaction Seminar](httsp://web.fhnw.ch/hgk/projekte/interaktion/)
+A project initiated in the [Interaction Seminar](https://web.fhnw.ch/hgk/projekte/interaktion/).
 
-ludwig.zeller -a-t- fhnw.ch
+Current development: [Benedikt Groß](httsp://benedikt-gross.de), [Fabian Morón Zirfas](https://fabianmoronzirfas.me), [Ted Davis](https://teddavis.org), [Timo Rychert](http://timorychert.de)  
```

Comment:
It's https://benedikt-gross.de not httsp…

Reply:
aha - no idea how/where this httsp was introduced?? i corrected it on a different and didn't even pay attention to if others had the mistake - which a few do (now searching)...

Reply:
solved with #141